### DISCUSSION TITLE
Got this error while run in ios iphone 8

### DISCUSSION THREAD

**Discussion by ghost** [axmolengine/axmol#729] [General]
![Screenshot 2022-07-07 at 4 28 00 PM](https://user-images.githubusercontent.com/105281914/177758362-b229305c-a582-46db-b6e2-27625abd6ff4.png)
![Screenshot 2022-07-07 at 4 28 20 PM](https://user-images.githubusercontent.com/105281914/177758367-746e3576-bb56-48c2-b1dc-46a1f02ac699.png)
![Screenshot 2022-07-07 at 4 28 39 PM](https://user-images.githubusercontent.com/105281914/177758368-f25a1708-c6e4-432b-94ab-7d7ee8681aaa.png)
![Screenshot 2022-07-07 at 4 31 51 PM](https://user-images.githubusercontent.com/105281914/177758638-c4d51404-049d-44ff-aa32-199cd55f30f5.png)

 
please help me i got this error in appcontroller while i was run first time in xcode.

**Comment by halx99** [comment]
Can you provide detail information, such as
- engine revision
- reproduce steps

**Comment by ghost** [comment]
@halx99 thanks for your reply.  yes i was just made new project through following command and run directly on iphone simulator with xcode.

1) adxe new mygame -l cpp -p org.adxe.mygame
2)cmake -S . -B build -GXcode -DCMAKE_TOOLCHAIN_FILE=$ADXE_ROOT/cmake/ios.mini.cmake -DCMAKE_OSX_ARCHITECTURES=x86_64

that above steps i have followed. 


![Screenshot 2022-07-07 at 4 55 21 PM](https://user-images.githubusercontent.com/105281914/177762606-a78e3893-828b-47ed-81b7-64570aec2c5c.png)

**Reply by halx99** [reply]
I have checkout your step, it works correct:
```txt

Touch /Users/bytedance/dev/projects/mygame/build/bin/mygame/Debug/mygame.app (in target 'mygame' from project 'mygame')
    cd /Users/bytedance/dev/projects/mygame
    /usr/bin/touch -c /Users/bytedance/dev/projects/mygame/build/bin/mygame/Debug/mygame.app

** BUILD SUCCEEDED **
```

**Comment by ghost** [comment]
yes build was successfully created  but problem is that  i got above errors @halx99

**Reply by halx99** [reply]
![image](https://user-images.githubusercontent.com/6977319/177918427-c80327fd-4123-460d-b6cb-4c39d205aff5.png)

I just run success, please use branch `release` or checkout to the latest pre-release tag: `v1.0.0b8`.

**Comment by ghost** [comment]
okay @halx99 i am doing again. still i am using ADXE name in my root directory so that cause any problem ? should i have to change ??

**Reply by halx99** [reply]
Name migrate not ready, you still needs use `adxe` with branch `release` or tag `v1.0.0b8`.
